oracle-linux-image-tools/bin/provision.sh

```diff
@@ -2,7 +2,7 @@
 #
 # Packer main provisioning script
 #
-# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 1982-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
1982 was a fantastic year for bash scripts, apparently. :)

Reply:
Updated -- bf5ca17